JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -216,16 +216,30 @@ private fun TabContent(
     val selectedTabIndex = tabContent.indexOfFirst { it.section == currentSection }
     Column {
         TabRow(
-            selectedTabIndex = selectedTabIndex
+            selectedTabIndex = selectedTabIndex,
+            backgroundColor = MaterialTheme.colors.onPrimary,
+            contentColor = MaterialTheme.colors.primary,
+
         ) {
             tabContent.forEachIndexed { index, tabContent ->
+                val colorText = if (selectedTabIndex == index) MaterialTheme.colors.primary else MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
```

Comment:
Long line, not sure why the linter allowed it...

Reply:
I just updated line 225